sources/packages/api/src/esdc-integration/msfaa-integration/models/msfaa-integration.model.ts

```diff
@@ -62,7 +62,7 @@ export enum TransactionCodes {
 /**
  * Codes used to define if the MSFAA was signed or canceled/rejected.
```

Comment:
I know canceled it right, can we please change it to cancelled, as we mostly follow the British english in our enums and comments?

Reply:
I was not aware that we were following it internally in the code. Enrollment vs enrolment is a good example that we have not been following one or another at all.
If we are doing it, let's ensure that all devs have the spell checker configured to only British English.
@ann-aot @dheepak-aot, can we change the spell checker to validate only British English?

Reply:
Changed!

Reply:
are we following British English? I heard that Canadian English is a mix of British English and American English.

Reply:
I changed to cancelled because almost every other place was using it as "cancelled".